theme/n21/views/addons/addons.hbs

```diff
@@ -3,11 +3,13 @@
         <link rel="stylesheet" href="/styles/addons/addons.css"/>
     {{/content}}
     {{#content "page"}}
+      {{#ifConfig "FEATURE_EXTENSION_PORTFOLIO_ENABLED" true}}
```

Comment:
could it be
`{{#if FEATURE_EXTENSION_PORTFOLIO_ENABLED }}` ?

Reply:
This needs to resolve the configuration, not an if: https://github.com/schul-cloud/schulcloud-client/blob/dcc73af86919fb7f0d665648be0546e90dd29aa9/helpers/handlebars/helpers/index.js#L139

Reply:
ok :)

Reply:
No, this needs to resolve https://github.com/schul-cloud/schulcloud-client/blob/dcc73af86919fb7f0d665648be0546e90dd29aa9/helpers/handlebars/helpers/index.js#L139